zScanner/ViewControllers/Documents/DocumentsListViewController.swift

```diff
@@ -83,12 +83,12 @@ class DocumentsListViewController: BaseViewController, ErrorHandling {
                     self.rightBarButtons = []
                     
                 case .loading:
-                    self.rightBarButtons = [self.loadingItem]
+                    self.rightBarButtons = []
                     
                 case .success:
                     self.rightBarButtons = []
                     self.departmentsStackView.subviews.forEach({ ($0 as? UIButton)?.isSelected = false })
-                    self.coordinator.createNewDocument()
+                    self.coordinator.createNewDocument(with: self.documentsViewModel.lastSelectedDepartment!)
```

Comment:
It's better to unwrap it. If you don;t really have to, don;t use force unwrapping. 
```
if let department = self.documentsViewModel.lastSelectedDepartment {
    self.coordinator.createNewDocument(with: department)
}
```